tests/pulser_simulation/test_qutip_backend_v2.py

```diff
@@ -0,0 +1,136 @@
+# Copyright 2025 Pulser Development Team
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from __future__ import annotations
+
+import math
+
+import numpy as np
+import pytest
+import qutip
+
+import pulser
+from pulser.backend.default_observables import Energy, Occupation, StateResult
+from pulser_simulation.qutip_backend import QutipBackendV2
+from pulser_simulation.qutip_config import QutipConfig
+from pulser_simulation.simconfig import SimConfig
+from pulser_simulation.simulation import QutipEmulator
+
+Omega_max = 4 * 2 * math.pi
+U = Omega_max / 2
+delta_0 = -6 * U
+delta_f = 2 * U
+t_rise = 500
+t_fall = 1000
+t_sweep = (delta_f - delta_0) / (2 * np.pi * 10) * 1000
```

Comment:
nit: Not sure you need to make these global. I would move them inside the `sequence` fixture